spec/unit/hathifiles_redirects_verifier_spec.rb

```diff
@@ -0,0 +1,127 @@
+# frozen_string_literal: true
+
+require "verifier/hathifiles_redirects_verifier"
+require "zinzout"
+
+module PostZephirProcessing
+  RSpec.describe(HathifileRedirectsVerifier) do
+    let(:test_date) { Date.parse("2023-01-01") }
+    let(:verifier) { described_class.new(date: test_date) }
+    let(:redirects_file) { verifier.redirects_file(date: test_date) }
+    let(:redirects_history_file) { verifier.redirects_history_file(date: test_date) }
+    # Including this mess should invalidate either file
+    let(:mess) { "oops, messed up a line!" }
+
+    # Clean dir before each test
+    before(:each) do
+      [ENV["REDIRECTS_DIR"], ENV["REDIRECTS_HISTORY_DIR"]].each do |dir|
+        FileUtils.rm_rf(dir)
+        FileUtils.mkdir_p(dir)
+      end
+    end
+
+    around(:each) do |example|
+      with_test_environment { example.run }
+    end
+
+    def stage_redirects_file
```

Comment:
don't need to stage if we are using climate control

Reply:
I'm still staging files in a temp dir so I can mess with them.